nuxt/components/product-stage.vue

```diff
@@ -1,143 +1,133 @@
+        fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path opacity="0.0591615" fill-rule="evenodd" clip-rule="evenodd" d="M0 0L316 695L1441 965L0 840V0Z"
+          fill="url(#paint0_linear)" />
+        <path opacity="0.0339753" fill-rule="evenodd" clip-rule="evenodd" d="M747 147L1439 965L1440 0H340L747 147Z"
+          fill="black" />
+        <path opacity="0.174464" fill-rule="evenodd" clip-rule="evenodd" d="M646 0L1261 222L1368 685L628 890L646 0Z"
+          fill="white" />
         <defs>
-          <linearGradient
-            id="paint0_linear"
-            x1="-327.295"
-            y1="767.888"
-            x2="135.719"
-            y2="1396.04"
-            gradientUnits="userSpaceOnUse"
-          >
+          <linearGradient id="paint0_linear" x1="-327.295" y1="767.888" x2="135.719" y2="1396.04"
+            gradientUnits="userSpaceOnUse">
             <stop />
             <stop offset="1" stop-color="white" stop-opacity="0.01" />
           </linearGradient>
         </defs>
       </svg>
     </div>
 
-    <div class="product-stage__content container position-relative">
+    <div :class="stage.cutoff ? 'd-flex justify-content-center' : 'product-stage__content container position-relative'">
+      <div class="row position-relative justify-content-end" v-if="stage.cutoff">
+      <div class="product-stage__content col col-md-6 px-10">
+        <div
+        :class="['space-top-2 space-top-lg-4 space-top-xl-5 space-bottom-1 space-bottom-lg-2', { 'text-white': light }]">
+        <headline :classes="stage.headlineClasses" :level="stage.headlineLevel || 'h1'">{{ stage.headline }}</headline>
+        <p v-if="stage.description" class="lead mt-5 px-0" v-html="stage.description"></p>
+        </div>
+        <pricing-slider v-if="stage.slider && products" :slider="stage.slider" :tooltip="stage.tooltip"
+        :modal-id="stage.modalId" :products="products" :light="light" />
+        <modal v-else-if="stage.modalId && stage.tooltip" :slim="true" :center="true" class="pricing-slider__modal"
+        :modal-id="stage.modalId">
+        <div class="container">
+          <div class="row">
+          <div class="col" v-html="stage.tooltip"></div>
+          </div>
+        </div>
+        </modal>
+        <div v-if="stage.additionalCopy" class="row justify-content-center py-6">
+        <div :class="['text-center', { 'text-white': light }]" v-html="stage.additionalCopy"></div>
+        </div>
+        <div v-if="stage.buttons" class="row justify-content-center py-6">
+        <cta-list :list="stage.buttons" />
+        </div>
+        <div v-else class="space-bottom-2 space-bottom-lg-3"></div>
+        <ul v-if="cards" class="row nav nav-pills" :class="{ 'px-1 px-lg-0': cards.tabs }" role="tablist">
+        <li v-for="(card, index) in cards.list" :key="index" :class="cardWrapperClasses">
+          <conditional-link :link="hasLink(card)" :href="card?.link?.href" :target="card?.link?.target"
```

Comment:
You can't do that when there is a cta inside. Otherwise you wrap an <a> tag around an <a> tag which might not run into an error but is semantically incorrect.

What is the purpose of the link around everything?